angular.json

```diff
@@ -35,8 +35,7 @@
                 "output": "/assets"
               },
               "src/favicon.ico",
-              "src/assets",
-              "src/web.config"
+              "src/assets"
```

Comment:
This is a different change, that's probably been merged in - it's fine to leave it in